http-clients/feign-common/src/test/java/org/triplea/http/client/GenericServerResponseTest.java

```diff
@@ -0,0 +1,33 @@
+package org.triplea.http.client;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import org.junit.jupiter.api.Test;
+
+class GenericServerResponseTest {
+
+  @Test
+  void builder() {
+    GenericServerResponse genericServerResponse = GenericServerResponse.builder().build();
+    assertFalse(genericServerResponse.isSuccess());
+    assertNull(genericServerResponse.getMessage());
+  }
+
+  @Test
+  void isSuccess() {
+    GenericServerResponse genericServerResponseTrue =
+        GenericServerResponse.builder().success(true).build();
+    GenericServerResponse genericServerResponseFalse =
+        GenericServerResponse.builder().success(false).build();
```

Comment:
Take a second look for if we need the 'failure' test case. We might be dangerously close to testing lombok, though it seems this case is already handled in the `builder` case above. The only difference between this one and the other is seemingly library code and JVM defaults (which we don't really need to test necessarily).

Reply:
I was going for coverage but I see your point that the builder would handle it. I'll remove it.